requirements/extra.txt

```diff
@@ -7,4 +7,4 @@ omegaconf>=2.0.5
 hydra-core>=1.0.5
 jsonargparse[signatures]>=4.3.0
 gcsfs>=2021.5.0
-rich>=10.2.2
+rich>=10.2.2,!=10.15.0,!=10.15.1
```

Comment:
Rather than avoiding, just have the minimal version?

Reply:
how about?
```suggestion
rich>=10.2.2,!=10.15.*
```